### PR TITLE
config: option for overrides to chroma styles

### PR DESCRIPTION
**Pull request by ktprograms**
User facing changes:
- Remove file option from config
- Move theme option to theme.name
- Add theme.chromaStyleOverrides map for style overrides

Code changes:
- Copy code from quick.Highlight to use chroma.Style object instead of looking up by name
- Pass chroma.Style pointer instead of object as chroma Format requires style pointer

Fix: #27 
Fix: #30

### CODE REVIEW

**Comment by noahgorstein** [comment]
Thanks for your PR @ktprograms. I'll be out of town without my computer for about a week but will try to get to this when I'm back later next week. Cheers.

File: README.md [hunk 1]
```diff
@@ -97,20 +97,36 @@ If a configuration option is present in both the configuration file and the comm
...
 
+### Chroma Style Overrides
+
+Overrides to the chroma styles used for a theme can be configured in your [configuration file](#configuration). 
+
+For the list of short keys, see [`chroma.StandardTypes`](https://github.com/alecthomas/chroma/blob/d38b87110b078027006bc34aa27a065fa22295a1/types.go#L210-L308). To see which token to use for a value, see the [JSON lexer](https://github.com/alecthomas/chroma/blob/master/lexers/embedded/json.xml) (look for `<token>` tags). To see the color and what's used in the style you're using, look for your style in the chroma [styles directory](https://github.com/alecthomas/chroma/tree/master/styles).
+
+```yaml
+theme:
+  name" "monokai" # name is required to know which theme to override
```
Comment (noahgorstein) [inline README.md:125]:
```suggestion
  name: "monokai" # name is required to know which theme to override
```

**Review by noahgorstein — CHANGES_REQUESTED** [review]
This looks and works great @ktprograms. Small typo in the README but good to go otherwise. Thanks again for your contribution and apologies for just getting around to review this.

File: README.md (hunk 1 above)
Comment (ktprograms) [reply README.md]: Oops, fixed (and also rebased on `main`)

**Review by noahgorstein — APPROVED** [review]

**Pull request merged**